packages/lesswrong/components/comments/DebateResponse.tsx

```diff
@@ -0,0 +1,222 @@
+    const readerIsParticipant = currentUser && fullParticipantSet.has(currentUser._id);
+
+    const showHeader = isFirstCommentInBlock;
+    const showInlineReplyForm = isLastCommentInBlock && !readerIsParticipant;
+    const showReplyLink = replies.length > 0 || showInlineReplyForm;
+    const addBottomMargin = isLastCommentInBlock;
+    const borderStyle = getParticipantBorderStyle(commentParticipantIndex);
+
+    const header = showHeader && <>
+      <CommentUserName comment={comment} className={classes.username} />
+      <span>{" " /* Explicit space (rather than just padding/margin) for copy-paste purposes */}</span>
+      <CommentsItemDate comment={comment} post={post} />
+    </>;
+
+    const menu = <CommentsMenu
+      comment={comment}
+      post={post}
+      showEdit={() => showEditForResponse(true, idx)}
+      className={classes.menu}
+    />;
+
+    const votingSystemName = comment.votingSystem || "default";
+    const votingSystem = getVotingSystemByName(votingSystemName);
+    const voteProps = useVote(comment, "Comments", votingSystem);
+    const commentItemRef = useRef<HTMLDivElement|null>(null); // passed into CommentsItemBody for use in InlineReactSelectionWrapper
+
+    const VoteBottomComponent = votingSystem.getCommentBottomComponent?.() ?? null;
+
+
+    const commentBodyOrEditor = showEdit[idx]
+    ? <CommentsEditForm
+        comment={comment}
+        successCallback={() => showEditForResponse(false, idx)}
+        cancelCallback={() => showEditForResponse(false, idx)}
+        className={classes.editForm}
+        formProps={{ post }}
+      />
+    : (<div ref={commentItemRef}>
+        <CommentBody comment={comment} voteProps={voteProps} commentItemRef={commentItemRef}/>
+      </div>
+    );
+
+    const replyLink = showReplyLink && <a className={classNames("comments-item-reply-link", classes.replyLink)} onClick={e => showRepliesForComment(e, idx)}>
+      Reply <span>({replies.filter(replyComment => replyComment.topLevelCommentId === comment._id).length})</span>
+    </a>;
+
+    const replyCommentList =
+      <DebateCommentsListSection
+        comments={replies}
+        totalComments={replies.length}
+        post={post}
+        newForm={showInlineReplyForm}
+        newFormProps={{
+          parentComment: comment,
+          replyFormStyle: 'minimalist',
+        }}
+      />;
+
+    const replyState = showReplyState[idx] && showReplyLink && replyCommentList;
+
+    return (
+      <div
+        key={`debate-comment-${comment._id}`}
+        id={`debate-comment-${comment._id}`}
+        className={classNames(classes.innerDebateComment, classes[borderStyle], { [classes.blockMargin]: addBottomMargin }, classes.lwReactStyling)}
+      >
+        {header}
+        {menu}
+        <div className={classes.commentWithReplyButton}>
+          {commentBodyOrEditor}
+          {replyLink}
+        </div>
+        {replyState}
+        {VoteBottomComponent && <div className={classes.reacts}>
```

Comment:
See previous comment?

Reply:
Ray discussed this in slack and suggested holding off for now and fixing if and when users start to complain.